docs/rst/manual/about/release_notes.rst

```diff
@@ -1,6 +1,120 @@
 Release Notes
 =============
 
+Version 2.6
+-----------
+
+Highlighted changes
+~~~~~~~~~~~~~~~~~~~
+
+Improvements to ERT CLI
+#######################
+The text and shell interface has been removed from ERT, but the CLI interface has gotten a upgrade and now
+included basic monitoring to show the progress of the running experiment. The CLI now also supports MDA and
+running single workflows.
+
+The see the usage and complete list of available commands, go to :doc:`../reference/running_ert`.
+
+
+Improvements to ERT GUI
+#######################
+The ERT GUI now includes help links to relevant resources and the job monitoring window now also includes
+memory usage information for each job. In addition, the output from the Eclipse job is treated like any
+other job and you can now read it from the the job monitoring window.
+
+Experimental features
+#####################
+The new iterative ensemble smoother algorithm has been added as an update algorithm.
+
+
+2.6 ERT application
+~~~~~~~~~~~~~~~~~~~
+
+New features:
+  - Add basic monitoring to ERT cli
+  - Memory usage monitoring in GUI
+  - CLI supports MDA
+  - CLI supports running single workflow
+
+Improvements:
+  - Less resource intensive monitoring
+  - Display config file name in all GUI window titles
+  - Run dialog no longer closes when pressing esc
+  - Improved exit monitoring when simulations fail
+  - Active realizations field is automatically filled with the runable realizations available
+  - Tailored plotting for single data points
+  - Algorithm recommendations in menu
+  - Check for observation data
+  - Better support for lsf-nodes with unknown status
+  - Communicate analysis completetion
+  - Various improvements to the documentation
+  - Document RMS job
+  - Help links in the GUI
+
+Code structure and tooling:
+  - Removed all C-code and CMake
+  - Removed ERT_SHARE_PATH
+  - Make CLI independent of Qt
+
+Deprecation:
```

Comment:
We have however deprecated a bunch of keywords and workflows, perhaps we should add those here?

Reply:
I see those are mentioned under the breaking changes section :clap: